xml/obs_staging_workflow.xml

```diff
@@ -0,0 +1,123 @@
+    &lt;staged_requests count="0"/&gt;
+    &lt;untracked_requests count="0"/&gt;
+    &lt;requests_to_review count="0"/&gt;
+    &lt;obsolete_requests count="0"/&gt;
+    &lt;missing_reviews count="0"/&gt;
+    &lt;broken_package count="0"/&gt;
+    &lt;checks count="0"/&gt;
+    &lt;missing_checks count="0"/&gt;
+  &lt;/staging_project&gt;
+  &lt;staging_project name="home:Admin:Staging:B" state="empty"&gt;
+    &lt;staged_requests count="0"/&gt;
+    &lt;untracked_requests count="0"/&gt;
+    &lt;requests_to_review count="0"/&gt;
+    &lt;obsolete_requests count="0"/&gt;
+    &lt;missing_reviews count="0"/&gt;
+    &lt;broken_package count="0"/&gt;
+    &lt;checks count="0"/&gt;
+    &lt;missing_checks count="0"/&gt;
+  &lt;/staging_project&gt;
+&lt;/staging_projects&gt;
+    </screen>
+  </sect2>
+
+  <sect2 xml:id="sec.obs.staging.workflow.projects.show">
+    <title>Overview of all staging projects</title>
+    <para>
+      This endpoint provides an overview of a single staging project.
+    </para>
+    <screen>
+geeko > <command>osc</command> api '/staging/openSUSE:Factory:Staging/staging_projects/openSUSE:Factory:Staging:A'
+    </screen>
+
+    <para>
+      Which will return the following XML:
+    </para>
+
+    <screen language="xml">
+&lt;staging_project name="home:Admin:Staging:A" state="empty"&gt;
+  &lt;staged_requests count="0"/&gt;
+  &lt;untracked_requests count="0"/&gt;
+  &lt;requests_to_review count="0"/&gt;
+  &lt;obsolete_requests count="0"/&gt;
+  &lt;missing_reviews count="0"/&gt;
+  &lt;broken_package count="0"/&gt;
+  &lt;checks count="0"/&gt;
+  &lt;missing_checks count="0"/&gt;
+&lt;/staging_project&gt;
+    </screen>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec.obs.staging.workflow.requests">
+  <title>Working with requests</title>
+  <para>One of the main features of the staging workflow is assigning incoming requests to different staging projects which is described in this section.</para>
+
+  <sect2 xml:id="sec.obs.staging.workflow.assign.request">
+    <title>Assign requests into a staging project</title>
+    <para>
+      Our main project openSUSE:Factory received requests with number 1 and 2. 
+      We would like to group these two requests together and move them into the staging project openSUSE:Factory:Staging:A.
+      This can be done with the following command which will create a link of the source package in openSUSE:Factory:Staging:A.
+    </para>
+    <screen>
+geeko > <command>osc</command> api -X POST '/staging/openSUSE:Factory:Staging/staging_projects/openSUSE:Factory:Staging:A/staged_requests' -d '&lt;requests&gt;&lt;number&gt;1&lt;/number&gt;&lt;number&gt;2&lt;/number>&lt;/requests&gt;'
+    </screen>
+  </sect2>
+
+  <sect2 xml:id="sec.obs.staging.workflow.remove.request">
+    <title>Remove requests from a Staging Project</title>
+    <para>
+      When we are done testing with of the staging project openSUSE:Factory:Staging:A, we need to remove the requests 1 and 2 again.
```

Comment:
@ChrisBr "with of" is probably a left over

Reply:
other way around :smile: ``with testing of``